test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -451,6 +429,120 @@ func TestMultiplePolicyBindings(t *testing.T) {
 	checkFailureReason(t, err, metav1.StatusReasonInvalid)
 }
 
+func TestCRDParams(t *testing.T) {
+	testcases := []struct {
+		name          string
+		schema        string
+		crSpec        map[string]interface{}
+		policy        *admissionregistrationv1alpha1.ValidatingAdmissionPolicy
+		policyBinding *admissionregistrationv1alpha1.ValidatingAdmissionPolicyBinding
+		namespace     *v1.Namespace
+		err           string
+		failureReason metav1.StatusReason
+	}{
+		{
+			name: "a rule that uses data from a CRD param resource does NOT pass",
+			schema: `{
+				"type":"object",
+				"properties":{
+				   "spec":{
+					  "type":"object",
+					  "properties":{
+						 "someNum":{
+							"type":"integer"
+						 }
+					  }
+				   }
+				}`,
+			crSpec: map[string]interface{}{
+				"someNum": 3,
+			},
+			policy: withValidations([]admissionregistrationv1alpha1.Validation{
+				{
+					Expression: "params.spec.someNum == 2",
```

Comment:
Should the expression use the param object to compare it against the object being validated in some way?

Reply:
Maybe compare `someNum` against a Deployment's replicaset or similar?

Reply:
I changed it to use strings/a similar name check as with some of the other tests - it's a bit similar to the `ConfigMap` tests now, but I think the CRD codepaths are still being hit, so it should be fine.